### DISCUSSION TITLE
Spotify question

### DISCUSSION THREAD

**Discussion by Leoner35** [ShankarBUS/ModernFlyouts#776] [Q&A]
I have a problem and it is that I do not know how to make what I am listening to on spotify appear on the sound bar, such as what appears on the program's website.
If someone could help me I would really appreciate it.
Thanks  ;)

**Comment by prakharb5** [comment]
So, you cannot see the media flyout? Or do you not see song info in the media flyout? What issue do you face?

**Reply by Leoner35** [reply]
Yes, I put a song on spotify and nothing comes out.
Only the volume comes out and when I put Lock. Mayus, numb ...

**Reply by prakharb5** [reply]
Okay, seems like #710.
